<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplaySortedClass</class>
 <widget class="QMainWindow" name="DisplaySortedClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1699</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplaySorted</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QCheckBox" name="T_Alphabetical">
        <property name="text">
         <string>Sort Alphabetically</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_AFC">
        <property name="text">
         <string>American Football Conference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_NFC">
        <property name="text">
         <string>National Football Conference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_NFCNorth">
        <property name="text">
         <string>NFC North</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_Stadium">
        <property name="text">
         <string>Sort by Stadium Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_Open">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_Date">
        <property name="text">
         <string>Sort by Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_Capacity">
        <property name="text">
         <string>Sort by Capacity</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="T_Conference">
        <property name="text">
         <string>Sort by Conference</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QGroupBox" name="displayGroupBox">
        <property name="styleSheet">
         <string notr="true">QGroupBox{
	background: white;
	border-radius: 20px;
}
</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="teamLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Team</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="stadiumLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Stadium</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="capacityLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Capacity</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="locationLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Location</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="roofTypeLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Roof Type</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="surfaceLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Surface</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="openedLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Opened</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="conferenceLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Conference</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="divisionLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>12</pointsize>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>Division</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>3</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>1659</width>
              <height>701</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QVBoxLayout" name="vertical_details_layout"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>30</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1699</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
